draft-irtf-cfrg-spake2.xml

```diff
@@ -175,14 +179,14 @@
 
       <section title="SPAKE2" anchor="spake2">
         <t>To begin, A picks x randomly and uniformly from the integers in [0,p),
-            and calculates X=x*P and S=w*M+X, then transmits pA=S to B.</t>
+            and calculates X=x*P and pA=w*M+X, then transmits pA to B.</t>
```

Comment:
This seems less clear than what was there previously, so I'd personally revert this. But, 🤷‍♂️